sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -4,12 +4,17 @@
 
 ### Features Added
 
+- Added `SearchIndexClient.list_index_stats_summary`.
+- Added `SearchIndexerCache.id`.
+
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
 
```

Comment:
Probably worth calling out the default API version update here, like you've done in the past.